theano/sandbox/cuda/dnn.py

```diff
@@ -1752,10 +1760,13 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
         Subsampling stride (default: (1, 1)).
     mode : {'max', 'average_inc_pad', 'average_exc_pad}
     pad
-        (padX, padY) padding information.
-        padX is the size of the left and right borders,
-        padY is the size of the top and bottom borders.
-    :param nd: dimensions of pooling, can be 2 or 3 for 2d or 3d pooling
+    	(pad_h, pad_w) padding information.
+        pad_h is the number of zero-valued pixels added to each of the top and
+        bottom borders.
+        pad_w is the number of zero-valued pixels added to each of the left
+        and right borders.
+    nd
+    	Number of dimensions of pooling, can be 2 or 3 for 2d or 3d pooling
```

Comment:
I fixed this in another pr.
Le 7 oct. 2015 16:30, "abergeron" <notifications@github.com> a écrit :

> In theano/sandbox/cuda/dnn.py
> <https://github.com/Theano/Theano/pull/3504#discussion_r41442595>:
>
> > @@ -1752,10 +1760,13 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
> >          Subsampling stride (default: (1, 1)).
> >      mode : {'max', 'average_inc_pad', 'average_exc_pad}
> >      pad
> > -        (padX, padY) padding information.
> > -        padX is the size of the left and right borders,
> > -        padY is the size of the top and bottom borders.
> > -    :param nd: dimensions of pooling, can be 2 or 3 for 2d or 3d pooling
> > +    	(pad_h, pad_w) padding information.
> > +        pad_h is the number of zero-valued pixels added to each of the top and
> > +        bottom borders.
> > +        pad_w is the number of zero-valued pixels added to each of the left
> > +        and right borders.
> > +    nd
> > +    	Number of dimensions of pooling, can be 2 or 3 for 2d or 3d pooling
>
> dnn_pool doesn't take an nd parameter. Maybe it should, but it doesn't
> currently.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3504/files#r41442595>.
>